gw_eccentricity/gw_eccentricity.py

```diff
@@ -67,25 +67,29 @@ def measure_eccentricity(tref_in=None,
                          extra_kwargs=None):
     """Measure eccentricity and mean anomaly from a gravitational waveform.
 
-    Eccentricity is measured using the GW frequency omega22(t) = dphi22(t)/dt,
-    where phi22(t) is the phase of the (2, 2) waveform mode. We currently only
-    allow time-domain, nonprecessing waveforms. We evaluate omega22(t) at
-    pericenter times, t_pericenters, and build a spline interpolant
-    omega22_pericenters(t) using those points. Similarly, we build
-    omega22_apocenters(t) using omega22(t) at the apocenter times,
-    t_apocenters. Finally, eccentricity is defined using omega22_pericenters(t)
-    and omega22_apocenters(t), as described in Eq.(1) of arxiv:xxxx.xxxx. Mean
-    anomaly is defined using t_pericenters, as described in Eq.(2) of
-    arxiv:xxxx.xxxx.
+    Eccentricity is measured using the GW frequency omega22(t) =
+    dphi22(t)/dt, where phi22(t) is the phase of the (2, 2) waveform
+    mode. We currently only allow time-domain, nonprecessing waveforms. We
+    evaluate omega22(t) at pericenter times, t_pericenters, and build a
+    spline interpolant omega22_pericenters(t) using those data
+    points. Similarly, we build omega22_apocenters(t) using omega22(t) at
+    the apocenter times, t_apocenters.
 
-    FIXME ARIF: In the above text, fill in arxiv number when available. Make
-    sure the above Eq numbers are right, once the paper is finalized.
+    Using omega22_pericenters(t) and omega22_apocenters(t), we first
+    compute e_omega22(t), as described in Eq.(4) of arxiv:xxxx.xxxx. We
+    then use e_omega22(t) to compute the eccentricity egw(t) using Eq.(8)
+    of arxiv:xxxx.xxxx. Mean anomaly is defined using t_pericenters, as
+    described in Eq.(10) of arxiv:xxxx.xxxx.
+
+    FIXME ARIF: In the above text, fill in arxiv number when
+    available.
 
     To find t_pericenters/t_apocenters, one can look for extrema in different
     waveform data, like omega22(t) or Amp22(t), the amplitude of the (2, 2)
-    mode. Pericenters correspond to peaks, while apocenters correspond to
-    troughs in the data. The method option (described below) lets you pick
-    which waveform data to use to find t_pericenters/t_apocenters.
+    mode. Pericenters correspond to the peaks (local maxima), while apocenters
+    correspond to the troughs (local minima) in the data. The method option
```

Comment:
Just don't say peaks/troughs as you are having to define them right away. Just say local maxima and local minima.

Reply:
Done